third_party/tests/Google/Google.log

```diff
@@ -2971,12 +3049,22 @@ Processing: -cd chapter-16 -I../../../UVM/1800.2-2017-1.0/src/ ../../../UVM/1800
 
 [WRN:PP0113] ../../../UVM/1800.2-2017-1.0/src/macros/uvm_object_defines.svh:806:9: Unused macro argument "OP".
 
+=======
+>>>>>>> Update logs
```

Comment:
this looks like a merge conflict header.

Reply:
Fixed.